packages/desktop-client/src/components/reports/disabledList.ts

```diff
@@ -94,54 +103,54 @@ const modeOptions = [
   },
 ];
 
+export function disabledGraphList(
+  item: string,
+  newGraph: string,
+  type: 'disabledSplit' | 'disabledType',
+) {
+  const [graphList] = modeOptions.filter(d => d.description === item);
+  const [disabledList] = graphList
+    ? graphList.graphs.filter(e => e.description === newGraph) || []
+    : [];
+  return disabledList[type];
+}
+
+export function disabledLegendLabel(
+  item: string,
+  newGraph: string,
+  type: 'disableLegend' | 'disableLabel',
+) {
+  const [graphList] = modeOptions.filter(d => d.description === item);
+  const [disableLegendLabel] = graphList
+    ? graphList.graphs.filter(e => e.description === newGraph) || []
+    : [];
+  return disableLegendLabel[type];
```

Comment:
Same comment here

Reply:
I think you might actually be able to merge these three utils into a single one to reduce the duplication.